src/deprecated/indexation-permalink-warning-presenter.php

```diff
@@ -46,6 +52,9 @@ public function present() {
 	/**
 	 * Determines which text to show as reason for the indexation.
 	 *
+	 * @deprecated 15.1
```

Comment:
This deprecated method contains the filter `wpseo_indexables_indexation_alert`. This filter has not been reintroduced in another class in this PR. Was that a conscious decision? I think we should either reintroduce it or deprecate the filter.

Reply:
I overlooked this filter. I re-introduced it in the `get_notification_message` method in `Indexing_Notification_Integration`.